docs/src/modules/components/ComponentsApiContent.js

```diff
@@ -146,9 +165,9 @@ export default function ComponentsApiContent(props) {
           <Heading text="import" hash={`${componentNameKebabCase}-import`} level="h3" />
           <HighlightedCode
             code={`
-import ${pageContent.name} from '${source}/${pageContent.name}';
+import ${namedImportName} from '${namedImportPath}';
```

Comment:
Shouldn't this be the other way around?
`import X from 'X'` is the default import, while `import { X } from 'X'` is the named one.